incubator/hnc/config/webhook/webhook_patch.yaml

```diff
@@ -7,6 +7,11 @@ webhooks:
 - name: objects.hnc.x-k8s.io
   timeoutSeconds: 2
   sideEffects: None
+  namespaceSelector:
+    matchExpressions:
+    - key: hnc.x-k8s.io/excluded-namespace
+      operator: NotIn
+      values: ["true"]
```

Comment:
Maybe use `DoesNotExist` as the operator? See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements

Reply:
Seems there are only `in`, `notin`, or `exists` according to your link https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement? So `notin` is the best option?

Reply:
I think that's what's available on the command line. Look at the "Resources that support set-based requirements" section for what's available through the API; that's where I saw `DoesNotExist`.